docs/policy/service-migrations-spring-2018.md

```diff
@@ -1,25 +1,70 @@
-Service Changes 2018
-====================
+Service Migrations - Spring 2018
+================================
 
-As you may have heard, the Open Science Grid (OSG) is undergoing some changes to its fabric of services.
-Some services will be retired, most services will be migrated with little planned effect on sites,
-and some services will be migrated with greater effect on sites.
-This document is intended to guide OSG site administrators through these changes.
-The following sections list various OSG services and whether or not site administrator action is required.
+The Open Science Grid (OSG) is transitioning effort from Indiana, requiring a redistribution of support and services.
+Some services will be retired, most services will be migrated to other locations (with minimal expected sites impact),
+and some services will be migrated that will result in significant impact on sites.
+
+This document is intended to guide OSG site administrators through these changes, highlighting where the site administrator
+action is required.
 
 !!! note
-    There will be individual announcements prior to retirement or migration of any OSG service.
+    Specific service migrations will be announced prior to retirement or migration.
 
+We understand sites will have many questions regarding this transition.
 The OSG Technology team will be holding office hours for any questions or comments about these service 
-changes; see the [end of the page](#office-hours) for details.
+changes; see the [Office Hours section](#office-hours) for details.
+
+Site Checklist Summary
+----------------------
+
+The remainder of the document provides a service-by-service overview of site impacts.  In this section, we
+provide a short outline of the steps sites are recommended to take:
+
+1. Review all site configuration (yum repositories, CVMFS configurations) to ensure they are the latest from the
```

Comment:
What CVMFS configurations need to be updated? We should add an action item to the OASIS/CVMFS section with details